vars/spinInfra.groovy

```diff
@@ -48,8 +50,14 @@ def call(product, component, environment, tfPlanOnly, subscription, deploymentTa
 
         def builtFrom = env.GIT_URL ?: 'unknown'
 
-        pipelineTags = new TerraformTagMap([environment: Environment.toTagName(environment), changeUrl: changeUrl, managedBy: teamName, BuiltFrom: builtFrom, contactSlackChannel: contactSlackChannel, application: env.TEAM_APPLICATION_TAG, businessArea: env.BUSINESS_AREA_TAG ]).toString()
-        log.info "Building with following input parameters: common_tags='$pipelineTags'; product='$product'; component='$component'; deploymentNamespace='$deploymentNamespace'; environment='$environment'; subscription='$subscription'; tfPlanOnly='$tfPlanOnly'"
+
+        if (environment != 'sandbox' && environment != 'sbox') {
+          pipelineTags = new TerraformTagMap([environment: Environment.toTagName(environment), changeUrl: changeUrl, managedBy: teamName, BuiltFrom: builtFrom, contactSlackChannel: contactSlackChannel, application: env.TEAM_APPLICATION_TAG, businessArea: env.BUSINESS_AREA_TAG ]).toString()
```

Comment:
please refactor this so that it's not repeating the code.

extract the map to a variable and then append the tags if needed and then create the pipeline tags and do the log message